schema/attestation.graphqls

```diff
@@ -33,5 +42,56 @@ type Attestation {
   attestation is the data being attested to.
   """
   attestation: String!
+
+  """
+  type
+  """
+  type: String!
+
+  """
+  source
+  """
+  source: Address!
+
+  """
+  dataversion
+  """
+  dataVersion: String!
+
+  """
+  producer
+  """
+  producer: String
+
+}
+
+"""
+AttestationFilter holds the filter parameters for the attestation querys.
+"""
+input AttestationFilter {
+  """
+  Filter attestations by data version.
+  """
+  dataVersion: String
+
+  """
+  Filter attestations by source type.
+  """
+  producer: String
+
+  """
+  Filter attestations recorded prior to this timestamp.
+  """
+  recordedBefore: Time
+
+  """
+  Filter attestations recorded after this timestamp.
+  """
+  recordedAfter: Time
```

Comment:
recorded makes me think these times are when the data in the attestation was record. But they are actually when the attestation itself was created.